guides/v2.2/cloud/project/project-start.md

```diff
@@ -60,61 +47,96 @@ In Integration, Staging, and Production, *only* the following directories are wr
+
+-  **Node 1**: `/var/log/platform/1.yw1unoukjcawe/post_deploy.log`
+-  **Node 2**: `/var/log/platform/2.yw1unoukjcawe/post_deploy.log`
+-  **Node 3**: `/var/log/platform/3.yw1unoukjcawe/post_deploy.log`
+
+The log for each deployment appends to this file. Check the timestamps on log entries to verify and locate the logs you want for a specific deployment. The following is a condensed example of log output that you can use for troubleshooting:
+
+```terminal
+Re-deploying environment project-integration-ID
+  Executing post deploy hook for service `mymagento`
+    [2019-01-03 19:44:11] NOTICE: Starting post-deploy.  
+    [2019-01-03 19:44:11] INFO: Validating configuration  
+    [2019-01-03 19:44:11] INFO: End of validation  
+    [2019-01-03 19:44:11] INFO: Enable cron  
+    [2019-01-03 19:44:11] INFO: Create backup of important files.  
+    [2019-01-03 19:44:11] INFO: Backup /app/app/etc/env.php.bak for /app/app/etc/env.php was created.  
+    [2019-01-03 19:44:11] INFO: Backup /app/app/etc/config.php.bak for /app/app/etc/config.php was created.  
+    [2019-01-03 19:44:11] INFO: php ./bin/magento cache:flush --ansi --no-interaction   
+    [2019-01-03 19:44:32] INFO: Warming up failed: http://integration-id-project.us.magentosite.cloud/
+    [2019-01-03 19:44:32] NOTICE: Post-deploy is complete.  
+```
+{: .no-copy}
 
-The value of `<project ID>` depends on the project ID and whether the environment is Staging or Production. For example, with a project ID of `yw1unoukjcawe`, the Staging environment user is `yw1unoukjcawe_stg` and the Production environment user is `yw1unoukjcawe`.
+The deploy log contains start and stop messages for each hook, such as `Start deploy.` and `Deployment complete.`.
 
-For example, on the Staging environment for project `yw1unoukjcawe`, the deploy log is located at `/var/log/platform/yw1unoukjcawe_stg/post_deploy.log`.
+### Application logs
 
-For Production, you have a three node structure. Logs are available with specific information for that node. For example, on the Production environment for project `yw1unoukjcawe`, the deploy log is located at node 1 `/var/log/platform/1.yw1unoukjcawe/post_deploy.log`, node 2 `/var/log/platform/2.yw1unoukjcawe/deploy.log`, and node 3 `/var/log/platform/3.yw1unoukjcawe/post_deploy.log`.
+To review other application logs in Staging or Production, you can access and review those logs in `/var/log/platform/ProjectID`.
 
-Logs for all deployments that have occurred on this environment are appended to this file. Check the timestamps on log entries to verify and locate the logs you want for a specific deployment.
+For Pro plan Staging, the project ID has `_stg` at the end. For example, if you receive `500` errors in Staging and want to review the NGINX logs, you can log in using SSH to the Staging environment and locate the logs in `/var/log/platform/ProjectID_stg`. For Pro plan Production, there are three nodes to check for logs.
```

Comment:
you can use SSH to log in to the Staging environment ....  
locate the logs in the `/var/log/platform/ProjectID_stg` directory 

Last sentence makes me think I'm going to see the directory location of the logs for the the three nodes, but they aren't there. (I know you mentioned them in earlier section -- consider clarifying.)